src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,65 @@
 package pages;
-
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class HomePage extends BasePage {
+import java.time.Duration;
 
-    By userAvatarIcon = By.cssSelector("img.avatar");
+public class HomePage extends BasePage {
 
-    public HomePage( WebDriver givenDriver) {
+    static WebDriverWait wait;
```

Comment:
Please, remove code redundancies

For example

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

![image](https://user-images.githubusercontent.com/104761482/214243851-48413893-4464-48e9-bf4b-df6392bf9e99.png)

![image](https://user-images.githubusercontent.com/104761482/214243804-d1654695-6866-4820-bc9f-42550945e9ee.png)